features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx

```diff
@@ -94,20 +115,47 @@ const useStyles = makeStyles((theme) => ({
 function UploadTheme() {
     const classes = useStyles();
     const [themeFile, setThemeFile] = useState([]);
+    const [fileName, setFileName] = useState('');
     const [isFileAccepted, setIsFileAccepted] = useState(false);
-    const [isUplaodUnsuccessful, setIsUploadUnsuccessful] = useState(false);
-
+    const [isUploadUnsuccessful, setIsUploadUnsuccessful] = useState(false);
+    const intl = useIntl();
+    const restApi = new API();
 
     const onDrop = (acceptedFile) => {
         setIsFileAccepted(true);
         setThemeFile(acceptedFile[0]);
     };
 
+    useEffect(() => {
+        const tenantThemeContent = restApi.exportTenantTheme();
+        tenantThemeContent.then((response) => {
+            const tenantFileName = response.headers['content-disposition'].split('filename=')[1];
+            setFileName(tenantFileName.slice(1, -1));
```

Comment:
Why do we need to remove the first character?

Reply:
fileName comes as **filename="test1.com"** so need to remove the "